Sources/ProjectSpec/Scheme.swift

```diff
@@ -51,14 +51,20 @@ public struct Scheme: Equatable {
 
     public struct Build: Equatable {
         public var targets: [BuildTarget]
+        public var parallelizeBuildables: Bool
```

Comment:
Can we rename this to `parallelizeBuild` to match both Xcode UI and internal name